packages/workflow/src/internals.ts

```diff
@@ -416,9 +430,55 @@ export class State {
     [
       '__enhanced_stack_trace',
       (): EnhancedStackTrace => {
-        // const stacks = this.getStackTraces();
-        // const sourceMap = this.sourceMap;
-        return { locations: [], sources: {} };
+        const { sourceMap } = this;
+        const stacks = this.getStackTraces();
+        const sdkInfo: SDKInfo = { name: 'typescript', version: '' }; //TODO: provide version value
+        const sourceMapRecord: Record<string, FileSlice[]> = {};
+        const stackTraces: StackTrace[] = [];
+
+        stacks.forEach((stack) => {
+          const locationsPaths: FileLocation[] = [];
+
+          errorStackParser.parse({ name: '', message: '', stack }).forEach((parsedStackTraceLine) => {
+            if (parsedStackTraceLine.columnNumber && parsedStackTraceLine.lineNumber && parsedStackTraceLine.fileName) {
+              const fileLocation: FileLocation = {
+                column: parsedStackTraceLine.columnNumber,
+                line: parsedStackTraceLine.lineNumber,
+                filepath: parsedStackTraceLine.fileName,
+                context: parsedStackTraceLine.functionName,
+              };
+              locationsPaths.push(fileLocation);
+
+              const fileSliceLineOffset: number = Math.max(1, fileLocation.line - Math.floor(fileSliceSize / 2));
+              const fileContent: string = sourceMap.sourcesContent[sourceMap.sources.indexOf(fileLocation.filepath)];
+              const fileSliceContentArray: string[] = fileContent.split('\n');
```

Comment:
Let's not slice for now, as mentioned in the PR this will might up syntax highlighting in the UI.
If we want to optimize later we could start by sending only the top locations in each trace.

Reply:
I'd still split the files and make sure the line terminator is always `\n`, note that on Windows line terminator is `\r\n` so you'd need to factor that in when you split the contents.